README.md

```diff
@@ -49,14 +51,13 @@ The following code example shows how [`physx`](physx/) can be initialized.
 const PX_PHYSICS_VERSION: u32 = physx::version(4, 1, 1);
 let mut foundation = Foundation::new(PX_PHYSICS_VERSION);
 
-let mut physics = PhysicsBuilder::default()
-    .load_extensions(false) // switch this flag to load extensions during setup
```

Comment:
Hm, where is this flag set now?

Reply:
There is a `PhysicsFoundationBuilder` which has a `with_extensions` method.